src/site/xdoc/server/quick-start-cassandra.xml

```diff
@@ -138,9 +145,9 @@ Step 8: Manage
 8.1. Manage via james-cli
 
   usage: 
-<i>$ java -jar server/container/cli/target/james-server-cli-3.0.0-beta5-SNAPSHOT.jar -h 127.0.0.1 -p 9999</i>
+<i>$ java -jar server/container/cli/target/james-server-cli.jar -h 127.0.0.1 -p 9999</i>
 
-  Available commands:
+  Here is a list of common commands:
     adduser &lt;/username&gt; &lt;/password&gt;
```

Comment:
In https://github.com/linagora/james-project/pull/837 I propose to use this case:

```
AddUser <username> <passwd>
ListUsers
etc...
```

IMO it makes it more understandable, and it is valid as commands are not case sensitive.